Dockerfile

```diff
@@ -242,10 +242,6 @@ ENV BUILDKIT_RUN_NETWORK_INTEGRATION_TESTS=1 BUILDKIT_CNI_INIT_LOCK_PATH=/run/bu
 FROM integration-tests AS dev-env
 VOLUME /var/lib/buildkit
 
-# newuidmap & newgidmap binaries (shadow-uidmap 4.8.1-r0) shipped with alpine cannot be executed without CAP_SYS_ADMIN,
-# because the binaries are built without libcap-dev.
-# So we need to build the binaries with libcap enabled.
-# TODO: ask the Alpine upstream to enable libcap: https://github.com/moby/buildkit/issues/2038
```

Comment:
I'd still prefer to keep this comment. At least we should retain the issue link.

Reply:
ah, sorry. I thought this was solved when we reorganized stuff because I saw `libcap-dev` was installed. But we just replaced qemu with cross-compile. Will restore

Reply:
what is the config option that is different from https://git.alpinelinux.org/aports/tree/community/shadow/APKBUILD#n30 ? Or does it detect that `libcap-dev` is installed (should make it more explicit then)

Reply:
Yes, it detects libcap-dev when installed

Reply:
Sent a merge request to https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/25365
Sorry, I should have worked on this much earlier 😓